packages/core/admin/server/src/domain/permission/index.ts

```diff
@@ -14,24 +14,12 @@ import {
   curry,
   merge,
 } from 'lodash/fp';
-
-export type Permission = {
```

Comment:
I don't know if we have actual style guides about this, but I thought we used interfaces where possible

Reply:
@Marc-Roig i prefer interface, what about you?

Reply:
same, lets use interfaces where possible 👍

Reply:
Seeing this was causing issues I say lets move forward with this!
But I think we also need to discuss where to put and get types in the backend side.

I would discourage moving the type definition from domain if possible, the domain folder holds the source of truth of the permissions (and the other types), and should be the only one responsible of changing those types.

cc @jhoward1994